### PR TITLE
tidb-dashboard: add doc for creating SQL binding from statement history

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

add doc for creating SQL binding from statement history.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/12817
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-statement-details.md [hunk 1]
```diff
@@ -8,12 +8,54 @@ aliases: ['/docs/dev/dashboard/dashboard-statement-details/']
...
+- The overview of SQL statements, which includes the SQL template, the SQL template ID, the current time range of displayed SQL executions, the number of execution plans, the database in which the SQL statement is executed, and the fast plan binding feature (area 1 in the following figure).
 - The execution plan list: If a SQL statement has multiple execution plans, this list is displayed. Besides text information of execution plans, TiDB v6.2.0 introduces visual execution plans, through which you can learn each operator of a statement and detailed information more intuitively. You can select different execution plans, and the details of the selected plans are displayed below the list (area 2 in the following figure).
 - Execution detail of plans, which displays the detailed information of the selected execution plans. See [Execution plan in details](#execution-details-of-plans) (area 3 in the following figure).
 
 ![Details](/media/dashboard/dashboard-statement-detail-v620.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:15]: The image is not updated.

File: dashboard/dashboard-statement-details.md [hunk 2]
```diff
@@ -8,12 +8,54 @@ aliases: ['/docs/dev/dashboard/dashboard-statement-details/']
...
 
+## Fast plan binding
+
+Starting from v6.6.0, TiDB introduces the fast plan binding feature. You can quickly bind a SQL statement to a specific execution plan in TiDB Dashboard.
+
+### Usage
+
+#### Bind an execution plan
+
+1. Click the **Plan Binding** button. The **Plan Binding** dialog box is displayed.
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:25]:
```suggestion
1. Click **Plan Binding**. The **Plan Binding** dialog box is displayed.
```

File: dashboard/dashboard-statement-details.md [hunk 3]
```diff
@@ -8,12 +8,54 @@ aliases: ['/docs/dev/dashboard/dashboard-statement-details/']
...
+    ![Fast plan binding - popup - bound](/media/dashboard/dashboard-quick-binding-popup-bound.png)
+
+3. After the binding is dropped, you can see the **Not bound** label.
+
+    ![Fast plan binding - popup](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
+
+### Limitation
+
+Currently, the fast plan binding feature does not support the following types of SQL statements:
+
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:54]:
```suggestion
- Statements that are not `SELECT`, `DELETE`, `UPDATE`, `INSERT`, or `REPLACE`
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: dashboard/dashboard-statement-details.md [hunk 4]
```diff
@@ -8,11 +8,53 @@ aliases: ['/docs/dev/dashboard/dashboard-statement-details/']
...
+    ![Fast plan binding - popup - bound](/media/dashboard/dashboard-quick-binding-popup-bound.png)
+
+3. After the binding is dropped, you can see the **Not bound** label.
+
+    ![Fast plan binding - popup](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
+
+### Limitation
+
+Currently, the fast plan binding feature does not support the following types of SQL statements:
+- Statements that are not `SELECT`, `DELETE`, `UPDATE`, `INSERT`, or `REPLACE`
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:54]:
```suggestion

- Statements that are not `SELECT`, `DELETE`, `UPDATE`, `INSERT`, or `REPLACE`
```

**Review by lilyjazz — APPROVED** [review]

**Pull request merged**